tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -285,6 +272,48 @@ As of Godot 4.0, exporting .NET projects is supported for desktop platforms
 (Linux, Windows and macOS). Other platforms will gain support in future 4.x
 releases.
 
+Common errors
```

Comment:
How about naming it "Common pitfalls"? I feel like the word _error_ is not quite right, maybe it's just me.

Reply:
Yeah, way better.